posthog/api/feature_flag.py

```diff
@@ -59,6 +61,11 @@ class FeatureFlagSerializer(TaggedItemSerializerMixin, serializers.HyperlinkedMo
     experiment_set: serializers.PrimaryKeyRelatedField = serializers.PrimaryKeyRelatedField(many=True, read_only=True)
     features: serializers.SerializerMethodField = serializers.SerializerMethodField()
     usage_dashboard: serializers.PrimaryKeyRelatedField = serializers.PrimaryKeyRelatedField(read_only=True)
+    dashboards = serializers.PrimaryKeyRelatedField(
```

Comment:
keeping up with the theme^

```suggestion
    analytics_dashboards = serializers.PrimaryKeyRelatedField(
```